Source/Core/Models/CardScheme.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  CardScheme.swift
+//  
+//
+//  Created by Alex Ioja-Yang on 05/08/2022.
+//
+
+public enum CardScheme {
```

Comment:
```suggestion
@frozen public enum CardScheme {
```

Reply:
I am not a fan of using frozen as this may as well change and it would become a breaking change for consumers. Would work if we were defining comparisons (< = >) or other things that would never change, but card schemes do come and go (example: we're looking at CB)